pydis_site/templates/home/index.html

```diff
@@ -19,21 +19,21 @@ <h1 class="is-size-1">Who are we?</h1>
         <div class="column is-half">
           <p>
             We're a large community focused around the Python programming language.
-            We believe anyone can learn programming, and are very dedicated to helping
-            novice developers take their first steps into the world of code. We also
+            We believe anyone can learn to code, and are very dedicated to helping
+            novice developers take their first steps into the world of programming. We also
             attract a lot of expert developers who are seeking friendships, collaborators,
-            or looking to hone their craft by teaching and getting involved in the community.
+            or to hone their craft by teaching and getting involved with the community.
 
             <br/><br/>
 
-            We organise regular community events like code jams, open source hackathons,
-            seasonal events and community challenges. Through our sponsorships and with
-            help from donations, we are even able to award prizes to the winners of our events.
+            We organise regular community events such as code jams, open-source hackathons,
+            seasonal events and community challenges. Through our sponsorships, and with
```

Comment:
Serial comma goodness.

```suggestion
            seasonal events, and community challenges. Through our sponsorships, and with
```